src/test/acceptance/features/confirmation-code-navigation.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Confirmation code navigation
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  Once I have entered my confirmation code
+  If I change the contact details used to receive that code
+  I have to go through the send and enter code part of the application to receive a new code.
+
+  Background:
+    Given I am on the first page of the application
```

Comment:
Strange that this isn't just a part of `I complete the application with valid details, selecting to receive my confirmation code via text`, any reason why not, I',m, pretty sure it is for the other complete flow steps.

Reply:
I went along the same route as check details which uses `Given I am on the first page of the application`